src/components/interface/Button/ButtonGroup.js

```diff
@@ -18,7 +18,8 @@ const ButtonGroup = ({
   ...remainingProps
 }) => {
   const content = Children.toArray(children)
-    .map((child) => <li key={child.props.title}>{child}</li>);
+    // eslint-disable-next-line react/no-array-index-key
+    .map((child, index) => <li key={index}>{child}</li>);
```

Comment:
No other options?

Reply:
no, title is not mandatory on the button, and button can have the same text :/